### PR TITLE
Minor changes to make pipelines work properly

### PR DESCRIPTION
**Pull request by DonHaul**
Minor Fixes:
A few urls were not correct due
Upon dag activation, jobs were running for every day instead of for a given workflow
Added black as well
Added Readme

### CODE REVIEW

**Comment by DonHaul** [comment]
@drjova could you check this PR, it is related to https://github.com/inspirehep/backoffice/pull/45
Lmk if you wish me to separate it in multiple PRs as currently there are changes in different areas

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ The setup is done with docker-compose. In order to run use
 docker-compose up -d
 ```
 
+in the UI go to Admin>Connections and set:
+- backoffice_conn : http host.docker.internal:8000
```
Comment (drjova) [inline README.md]:
```suggestion
- backoffice_conn: http host.docker.internal:8000
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ The setup is done with docker-compose. In order to run use
...
+- inspire_connection: https://inspirebeta.net
+in the UI go to Admin>Variables and set:
+- backoffice_token	auth token from django for a superuser
```
Comment (drjova) [inline README.md:15]:
```suggestion
- backoffice_token	auth token from django for a superuser
```

File: dags/author/author_create/author_create_init.py [hunk 3]
```diff
@@ -46,7 +46,7 @@ def set_workflow_status_to_running(**context):
 
     @task()
     def create_author_create_user_ticket(**context: dict) -> None:
-        endpoint = "/tickets/create-with-template"
+        endpoint = "/api/tickets/create"
```
Comment (drjova) [inline dags/author/author_create/author_create_init.py:45]: why we change the api endpoint?

Comment (DonHaul) [reply dags/author/author_create/author_create_init.py:45]: due to a change in inspirehep that pascal has done that was not yet updatted

https://github.com/inspirehep/inspirehep/pull/3097/files#diff-147704379813427143b24ca6b9e81b711aae49900a7fe939dbc6d131c9961708L58

Equivalent implementation in next:
https://github.com/inspirehep/inspire-next/pull/4333/files

**Review by drjova — APPROVED** [review]
@DonHaul please squash all commits :)

**Comment by DonHaul** [comment]
Ran into some problems during the squash, and cant reopen PR,
all modifications can be found in https://github.com/inspirehep/workflows/pull/3

**Pull request closed without merging**